takeshape/takeshape-starter-shopify-nextjs
# Fix static rendering of pages

This always render page contents, even when Shopify hasn't loaded it. Instead, the Add to Cart button is disabled and the cart component does not render  until Shopify is available. 

[Preview Deployment](https://shape-shop-next-allanlasser.vercel.app)

## Test Steps

1. Open the preview deployment
2. Disable Javascript in your browser (in Chrome, open DevTools Settings and check "Disable Javascript" option) and reload the page.
3. Pages should fully render, except the cart should be hidden and the "Add to Cart" button should be disabled.
